environments/staging/postgresql.yml

```diff
@@ -46,3 +47,12 @@ dbs:
       pgbouncer_host: pgproxy0
       django_migrate: False
 
+pgpool_config:
+  redis0:
+    primary: rds_pg0
+    standbys:
+      - rds_pg0_replica
+    num_init_children: 59
+    max_pool: 5
+    database_redirect_preference_list: 'commcarehq:1(0.8)'
```

Comment:
I think this would always be something like:
```
*:primary(0.x)
```

Reply:
We could have a different one as well, no? For e.g. for ICDS UCR dbs, we could have.
```
`commcarehq_icds_aggregatedata:primary(0.1)1(0.25)2(0.25)3(0.2)4(0.2)`
```

Reply:
We could but I think in 99% of cases we'd have the same weighting for all standbys.

Reply:
Anyway, it's configurable, so both of them can be done, unless there is value in specifically restricting to `*:primary(0.x)` format only. Your [above comment](https://github.com/dimagi/commcare-cloud/pull/2453#discussion_r256732577) is relevant here too for not doing that.

Reply:
No value in restricting it, I was just commenting